packages/webapp/src/store/api/apiSlice.ts

```diff
@@ -61,7 +61,56 @@ import type {
   SensorReadings,
   IrrigationPrescription,
 } from './types';
-import i18n from '../../locales/i18n';
+
+/**
+ * These tags have endpoints that do not return farm specific data, and are not created by a farm
+ *
+ *  LiteFarm provides these data as defaults
+ */
+export const LibraryTags = [
+  'AnimalSexes',
+  'AnimalIdentifierTypes',
+  'AnimalIdentifierColors',
+  'AnimalMovementPurposes',
+  'AnimalOrigins',
+  'AnimalUses',
+  'AnimalRemovalReasons',
+  'DefaultAnimalBreeds',
+  'SoilAmendmentMethods',
+  'SoilAmendmentPurposes',
+  'SoilAmendmentFertiliserTypes',
+];
+
+/**
+ * These tags contain endpoints that return farm specific data
+ *
+ * These data should not persist when switching farms,
+ * or should be stored in a separate farm store.
+ */
+export const FarmTags = [
+  'Animals',
+  'AnimalBatches',
+  'CustomAnimalBreeds',
+  'CustomAnimalTypes',
+  'FarmAddon',
+  'IrrigationPrescriptions',
+  'SoilAmendmentProduct',
```

Comment:
Not related at all to this PR but some tags are in plural and some in singular, we should probably make it consistent